Makefile

```diff
@@ -606,24 +606,22 @@ $(HELM):
 # creates istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml
 # Ensure that values-$filename is present in install/kubernetes/helm/istio
 isti%.yaml: $(HELM)
+	cat install/kubernetes/templates/namespace.yaml > install/kubernetes/$@
```

Comment:
Should the target here be `istio%.yaml` instead of `isti%.yaml`?

Reply:
Actually it won't work for istio.yaml